spec/components/back_link_spec.rb

```diff
@@ -13,11 +13,11 @@ def component_name
 
   it "renders a back link correctly" do
     render_component(href: '/back-me')
-    assert_select ".gem-c-back-link[href=\"/back-me\"]", text: "Back"
+    assert_select ".govuk-back-link[href=\"/back-me\"]", text: "Back"
```

Comment:
I think the tests will pass without this change - might be better?

Reply:
Yes, they still pass without the change. I thought would be good to rely on `.govuk` selectors for consistency (we're currently querying elements using `.govuk` classes) since we're aiming to keep `.gem-c-back-link` for highlighting components, not I can roll that back if you think we should continue to use `.gem-c` selectors where possible.